backend/routers/index_router.py

```diff
@@ -16,3 +25,17 @@ async def healthcheck():
 async def read_main():
     logger.info("In root path")
     return {"msg": "Hello World"}
+
+
+@index_router.get("/get_pixel_watch", response_model=List[schemas.PixelWatch])
+def get_pixel_watch(db: Session = Depends(get_db)):
+    users = db.query(PixelWatch).all()
+    return users
+
+@index_router.post("/watch_my_pixel")
+async def watch_my_pixel(params: WatchPixelParams,
```

Comment:
You have to additionaly request & store in db path + row, because one lat long may have multiple tiles